### PR TITLE
adds text position validation

### PR DESCRIPTION
**Pull request by noahamac**

### CODE REVIEW

**Comment by noahamac** [comment]
This PR adds a single lined validation check before each of the label generators. This check compares the computed label X position with the left bound of the visualization, ensuring that the labels are always visible. 

Also removes a console.log from the container. 

image

Find this viz on the Side by Side dashboard: https://app.dev.looker.com/dashboards/710

image

File: bar-gauge/HorizontalGauge.js [hunk 1]
```diff
@@ -99,33 +99,39 @@ const drawHorizontal = (props) => {
 	
 	// GAUGE VALUE LABEL
 	if (props.value_label_type === 'value' || props.value_label_type === 'both' || props.value_label_type === 'dboth') {
-		svg.append('text')
+		label_pointer = svg.append('text')
```
Comment (thomasbanghart) [inline bar-gauge/HorizontalGauge.js:102]: Should `var` be here?

Comment (thomasbanghart) [inline bar-gauge/HorizontalGauge.js:102]: Should `var` be here?

Comment (noahamac) [reply bar-gauge/HorizontalGauge.js]: implicit declaration 🙅‍♂️

**Pull request merged**